style/icons.css

```diff
@@ -109,8 +109,7 @@
   width: 35px;
 }
 
-.jp-ToggleSwitch::before {
-  content: '';
+.jp-Toolbar-item .jp-ToggleSwitch svg {
```

Comment:
There is now a default `svg` added to the dom when creating a new `ToolbarButton`:

![image](https://user-images.githubusercontent.com/591645/72271790-d4cc3a80-3627-11ea-870b-e7263fefa469.png)

We might want to investigate using the new `JLIcon` at some point instead of manipulating the element CSS directly.

Reply:
Agreed.